python_modules/dagster/dagster_tests/general_tests/test_import.py

```diff
@@ -23,9 +23,14 @@ def test_import_perf():
     import_profile = result.stderr.decode("utf-8")
 
     # ensure expensive libraries which should not be needed for basic definitions are not imported
-    assert "grpc" not in import_profile
-    assert "sqlalchemy" not in import_profile
-    assert "upath." not in import_profile  # dont conflate with import of upath_io_manager
-
-    # one way to debug imports is to `pip install tuna` then run
-    # python -X importtime python_modules/dagster/dagster_tests/general_tests/simple.py &> /tmp/import.txt && tuna /tmp/import.txt
+    try:
+        assert "grpc" not in import_profile
+        assert "sqlalchemy" not in import_profile
+        assert "upath." not in import_profile  # dont conflate with import of upath_io_manager
+    except AssertionError as exc:
```

Comment:
instead of try catch , can we just iterate through a loop and check then print a message with all the expensive libraries we found ?

Reply:
Sure! Makes more sense